docs/concepts/login.md

```diff
@@ -1,2 +1,74 @@
 (login)=
-# {octicon}`sign-in` Login
\ No newline at end of file
+# {octicon}`sign-in` Login
+
+Finetuner, as part of the [Jina AI](https://jina.ai/) stack,
+uses [Jina AI Cloud](https://cloud.jina.ai/) for authentication.
```

Comment:
```suggestion
Finetuner uses [Jina AI Cloud](https://cloud.jina.ai/) for authentication.
```

Reply:
why it is suggested in such as way?

Reply:
"...as part of the [Jina AI](https://jina.ai/) stack..." is superfluous and buzzwordy.